docs/details/advisors/checks/Develop Security Checks

```diff
@@ -0,0 +1,260 @@
+The Security Threat Tool in PMM offers a set of checks that can detect common security threats, performance degradation, data loss and data corruption. 
+
+As a developer, you can create custom checks to cover additional use cases that are relevant to your specific database infrastructure. 
+
+
+## Check components
+A check is a combination of:
+
+- SQL query or MongoDB query document for extracting data from the database;
+- Python script for converting extracted data into check results. This is actually a [starlark-go](https://github.com/google/starlark-go) script – [an extension](https://github.com/google/starlark-go/blob/master/doc/spec.md#dialect-differences) of the [original Starlark language](https://docs.bazel.build/versions/main/skylark/language.html) that adds more imperative features from the Python. Script's execution environment is sandboxed; no I/O can be done from it.
+- Tiers. By default, PMM has access to a set of anonymous checks, which can be downloaded even if PMM is not connected to Percona Platform. As soon as you connect your PMM instance to Percona Platform, PMM can download registered checks, which offer more advanced database health information.​
```

Comment:
This field used only for checks created by Percona and it's more about our plans policy. Not sure that should be mentioned in the public documentation.

Reply:
Agree this is internal only from my point of view